services/seeders/20180410172245-default-channels.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.bulkInsert('channels', [
+        {
+          name: 'canary',
+          createdAt: Sequelize.fn('NOW'),
+          updatedAt: Sequelize.fn('NOW')
+        },
+        {
+          name: 'beta',
+          createdAt: Sequelize.fn('NOW'),
+          updatedAt: Sequelize.fn('NOW')
+        },
+        {
+          name: 'general',
+          createdAt: Sequelize.fn('NOW'),
+          updatedAt: Sequelize.fn('NOW')
+        }
+    ], {});
```

Comment:
Just food for thought, I guess you read it: https://www.chromium.org/getting-involved/dev-channel: 

> Stable channel: This channel has gotten the full testing and blessing of the Chrome test team, and is the best bet to avoid crashes and other issues.  It's updated roughly every two-three weeks for minor releases, and every 6 weeks for major releases.
Beta channel: If you are interested in seeing what's next, with minimal risk, Beta channel is the place to be.  It's updated every week roughly, with major updates coming every six weeks, more than a month before the Stable channel will get them.
Dev channel: Want to see what's happening quickly, then you want the Dev channel.  The Dev channel gets updated once or twice weekly, and it shows what we're working on right now.  There's no lag between major versions, whatever code we've got, you will get.  While this build does get tested, it is still subject to bugs, as we want people to see what's new as soon as possible.
Canary build: Canary builds are the bleeding edge. Released daily, this build has not been tested or used, it's released as soon as it's built.
Other builds: If you're absolutely crazy, you can download the latest working (and that's a very loose definition of working) build from download-chromium.appspot.com. You can also look for a more specific recent build by going to the Chromium continuous build waterfall, looking at the number near the top under "LKGR", and then going to this Google Storage bucket and downloading the corresponding build.

Reply:
More specifically I think renaming `general` to `stable` over time might indeed convey a clearer message.

Reply:
@batmat I am aware of Chrome's stable channel, and I explicitly reject the label :smile: 

We are not exactly following Chrome's model because Chrome's model is not continuous, it's kind of a blend between a train and rolling release. The use of `stable` gives an implication that everything else is `unstable`, which is not the plan at all here. For something to get into the `stable` channel it must be blessed by a manual test team which we will never have for Jenkins Essentials (which isn't a bad thing).